docs/backends/azure.rst

```diff
@@ -170,3 +170,7 @@ The following settings are available:
     basis, subclass the backend and override ``AzureStorage.get_object_parameters``.
     
     This is a Python ``dict`` and the possible parameters are: ``content_type``, ``content_encoding``, ``content_language``, ``content_disposition``, ``cache_control``, and ``content_md5``.
+
+``AZURE_API_VERSION``
+
+    The api version to use. The defualt value is ``latest``.
```

Comment:
But does this even work?  the SDK has a list of supported API versions baked into it, and 'latest' is not among them:
```
>>> client = BlobServiceClient("https://foo.bar.com", api_version="latest")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/dch/.virtualenvs/foo/lib/python3.8/site-packages/azure/storage/blob/_blob_service_client.py", line 139, in __init__
    self._client._config.version = get_api_version(kwargs)  # pylint: disable=protected-access
  File "/home/dch/.virtualenvs/foo/lib/python3.8/site-packages/azure/storage/blob/_serialize.py", line 144, in get_api_version
    raise ValueError("Unsupported API version '{}'. Please select from:\n{}".format(api_version, versions))
ValueError: Unsupported API version 'latest'. Please select from:
2019-02-02
2019-07-07
2019-10-10
2019-12-12
2020-02-10
2020-04-08
2020-06-12
2020-08-04
2020-10-02
2020-12-06
2021-02-12
2021-04-10
```

Reply:
Thanks, I did notice 'latest' is not in the current sdk available on github. Possibly the better option is to omit AZURE_API_VERSION if undefined and insert if defined. Thoughts...?

Reply:
Yes, if you don't pass the parameter to the SDK then it will default to using the most recent version in its hardcoded list - which is almost always want you'd want, I expect.

Reply:
you could build a dictionary containing the arguments and then provide them as `BlobServiceClient(**kwargs)`